AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -27,51 +27,71 @@ final class CallingContext {
         self.tokenFetcher = tokenFetcher
     }
 
-    // MARK: Public API
+    // MARK: Private function
+    private func fetchInitialToken(completionHandler: @escaping (String?) -> Void) {
```

Comment:
Can we make these `async` methods to follow the rest of the authentication handling?